examples/azure_file/azure-key.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: azure-key
+type: Opague
+data:
+  key: Zm9vIC1uCg==
```

Comment:
For clarity, change `key` to `azureAccountKey`.
Move the account name here as `azureAccountName` (from API).
Document in `examples/azure_file/README.md` that the secret for Azure must be defined like this.